doozer/doozerlib/cli/get_nightlies.py

```diff
@@ -243,6 +269,16 @@ def rc_api_url(tag: str, arch: str) -> str:
     """
     arch = go_arch_for_brew_arch(arch)
     arch_suffix = go_suffix_for_arch(arch)
+    if private_nightly:
+        arch = f"-{arch}"
+        if arch == "-amd64":
+            # We do not need to add the arch to URL for amd64
+            return f"{constants.RC_BASE_PRIV_URL.format(arch='')}/api/v1/releasestream/{tag}-priv"
+
+        # For other arches, add it to the URL
+        return f"{constants.RC_BASE_PRIV_URL.format(arch=arch)}/api/v1/releasestream/{tag}{arch_suffix}-priv"
```

Comment:
```suggestion
        return f"{constants.RC_BASE_PRIV_URL.format(arch=arch)}/api/v1/releasestream/{tag}{arch_suffix}"
```

Reply:
sadly no - we would have to check for private nightly + amd64, and if true, we need to skip inserting the arch to the URL

Reply:
done